### PR TITLE
New Mentions Class and Mentions shown on mentions page as message followed by conversation name

### PR DESCRIPTION
**Pull request by Gjain234**
screen shot 2018-07-18 at 12 22 02 am

This is currently a bit ugly, but basically this mentions page is printing out the message and conversation attributes of the mentions arraylist inherent to the user. So for example, the first message I was tagged in was "@gaurijain tagged message 1" inside a conversation named "hello"

In order to implement this I have also added a mentions class, and I fixed some small things we were talking about yesterday in terms of entering the admin page from the mentions page. 

The chatservlet test is coming soon!

### CODE REVIEW

File: src/main/webapp/WEB-INF/view/mentions.jsp [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <%@ page import="codeu.model.store.basic.UserStore" %>
 <%@ page import="codeu.model.store.basic.ConversationStore" %>
 <%@ page import="codeu.model.store.basic.MessageStore" %>
+<%@ page import="codeu.model.data.User" %>
+<%@ page import="codeu.model.data.Mention" %>
+
```
Comment (ernesto-rojas) [inline src/main/webapp/WEB-INF/view/mentions.jsp:6]: Weird extra space?

**Review by ernesto-rojas — APPROVED** [review]
This looks good!

File: src/main/webapp/WEB-INF/view/mentions.jsp [hunk 2]
```diff
@@ -9,30 +12,32 @@
...
+      <% } %>
   </nav>
   <div id="container">
     <div style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
       <h1>Your Mentions</h1>
-      <p>You will show messages you were mentioned in here! (Not yet!)</p>
+      <p>You will show messages you were mentioned in here! <%
+      if(name!= null){
+        for (Mention x: UserStore.getInstance().getUser(name).getMentions()){
+        %><%=x.getMessage()%> <%=x.getConversation()%>
```
Comment (beckyqiu) [inline src/main/webapp/WEB-INF/view/mentions.jsp:37]: Add a new line here? Does "/n" works in this case? I am ok with this quick pull request but if it's a quick fix let's do it here. If it's not, it's fine to leave it for now.

Comment (Gjain234) [reply src/main/webapp/WEB-INF/view/mentions.jsp:37]: Hey Becky, as you can see it was already merged so I didn't have a chance to fix it, but we planned that Ernesto would fix the UI and display mentions on the jsp, so I think he will already find a better way to display it than this!

**Pull request merged**